vm-setup/roles/firewall/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+baremetal_tcp_ports: []
+baremetal_udp_ports:
+  - "6230"
+  - "6231"
+  - "6232"
+  - "6233"
+  - "6234"
+  - "6235"
```

Comment:
We should probably set this as a fact generated based on num_masters and num_workers - see https://github.com/openshift-metal3/dev-scripts/pull/830

Reply:
The range is configurable now, you'd be able to override this with `-e` in the ansible-playbook command 

```
VBMC_MAX_PORT=$((6230 + ${NUM_MASTERS} + ${NUM_WORKERS} - 1))
ansible-playbook [...]
   -e vmc_port_range="6230:$VBMC_MAX_PORT"
   [...]
```